### DISCUSSION TITLE
HomeKitTile als Popup ein Widget statt Gerätedetails aufrufen

### DISCUSSION THREAD

**Discussion by gimli1980** [Zefau/ioBroker.jarvis#2306] [Q&A] [answered]
Hallo zusammen,

ich bin gerade am Anfang mit meiner Visu und habe ein HomeKit Widget erstellt, welches mir anzeigt, ob alle Fenster geschlossen sind oder nicht.

Hierfür habe ich ein Skript im iobroker und einen Datenpunkt der mir true oder false liefert. Den Datenpunkt habe ich als Gerät in Jarvis eingefügt.

Jetzt hätte ich es gerne, dass mir beim Drücken auf das Widget ein zweites Widget geöffnet wird, in welchem mir der Status aller Fenster angezeigt wird. 

Aktuell löse ich es so, dass ich für die Fenster einen eigenen Tab hab, auf dem lediglich ein Widget, nämlich die statelist mit allen Fenstern ist. Das funktioniert auch soweit, ich fände es allerdings schicker wenn ich die statelist mit allen Fenstern als popup angezeigt bekommen würde.

Ist so etwas möglich?

LG Denny

**Comment by mcuiobroker** [comment, marked as answer]
Auch beim HomeKit Tile kann man ein Geräte-Popup einstellen.

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/v3-fuer-einsteiger/module/homekittile#aktion-beim-klick

Wenn man alle Fenster als Geräte DP eingestellt hat sieht man sie auch im Popup.

**Comment by gimli1980** [comment]
Vielen Dank für die schnelle Antwort! Hat einwandfrei geklappt.

Manchmal hat man einfach ein Brett vor dem Kopf.
